__tests__/app.test.js

```diff
@@ -65,18 +66,42 @@ describe("GET/api/categories", () => {
         .then(({ body }) => {
           expect(body).toEqual({
             review_id: 2,
-            title: expect.any(String),
-            review_body: expect.any(String),
-            designer: expect.any(String),
-            review_img_url: expect.any(String),
-            votes: expect.any(Number),
-            category: expect.any(String),
+            title: 'Jenga',
+            review_body: 'Fiddly fun for all the family',
+            designer: 'Leslie Scott',
+            review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+            votes: 5,
+            category: 'dexterity',
             owner: "philippaclaire9",
-            created_at: expect.any(String),
+            created_at: '2021-01-18T10:01:41.251Z',
           });
         });
     });
   })
+
+  describe('GET /api/reviews/:review_id/comments',()=>{
+    test.only('status:200, responds with an array of comments objects for the given review_id',()=>{
+      return request(app)
+      .get('/api/reviews/2/comments')
+      .expect(200)
+      .then((response)=>{
+        console.log(response.body, 'inside test');
+        const comments = response.body.comments
+        expect(comments ).toHaveLength(3); 
+        expect(comments).toBeSortedBy("comment_id", { descending: true })
+        comments.forEach((comment)=>{
+          expect.objectContaining({
+            comment_id:expect.any(Number),
+            votes: expect.any(String),
+            created_at:expect.any(String),
+            author:expect.any(String),
+            body: expect.any(String),
+            review_id:expect.any(String)
+          })
+        })
+      })
+    })
+  })
```

Comment:
Please add the following test:
Sad path testing 1.: review_is is the right datatype, but thee's no review with that ID. (404, 'not found')